crates/lib/src/structs.rs

```diff
@@ -117,28 +118,34 @@ pub enum MangaStatus {
 }
 
 #[repr(C)]
+#[derive(Clone, Copy, Debug, Default)]
 pub enum MangaContentRating {
+    #[default]
     Safe = 0,
     Suggestive = 1,
     Nsfw = 2,
 }
 
 #[repr(C)]
+#[derive(Clone, Copy, Debug, Default)]
 pub enum MangaViewer {
+    #[default]
     Default = 0,
```

Comment:
I'm thinking we should maybe remove this (the Default case) since sources should not return it and should attempt to provide the correct viewer instead.